src/components/FAQ.vue

```diff
@@ -180,6 +181,12 @@ export default {
       expandAll: false
     }
   },
+  computed: {
+    allItemsExpanded () {
+      const allQuestions = this.faqs.flatMap(block => block.qa)
+      return allQuestions.every(q => q.active)
+    }
```

Comment:
As opposed to the previous logic where the component has a flag that governs whether all items are expanded/collapsed, added a computed prop that checks it the whole time.